test_shaders.py

```diff
@@ -240,13 +259,22 @@ def test_shader_msl(stats, shader, update, keep):
     regression_check(shader, msl, update, keep)
     os.remove(spirv)
 
+def test_shader_hlsl(stats, shader, update, keep):
+    joined_path = os.path.join(shader[0], shader[1])
+    print('Testing HLSL shader:', joined_path)
+    spirv, msl = cross_compile_hlsl(joined_path)
+    regression_check(shader, msl, update, keep)
```

Comment:
Yes, if all text files are normalized to Unix-style line endings, it should be done on all text files and all OSes.